### DISCUSSION TITLE
[BUG] Add-PnPFile setting modified date 1 hour earlier than value supplied.

### DISCUSSION THREAD

**Discussion by Sharepoint2k22** [pnp/powershell#4620] [General]
### Reporting an Issue or Missing Feature

When using Add-PnPfile and setting a modified date for a file, the actual date set is 1 hour earlier

### Expected behaviour 

The time to be set exactly as supplied.

### Actual behaviour 

The time is set 1 hour earlier. To add that when using Set-PnPListItem to set the same item/time, the behaviour is correct.

### Steps to reproduce behaviour

#Add the file
Add-PnPFile -Path $localPath -Folder $Library -Values @{"Modified" = "05/28/2014 16:54"}

#Check the file value
$listitems = Get-PnPListItem -List $listName
$listitems[X].FieldValues.Modified:

28 May 2014 15:54:00

### What is the version of the Cmdlet module you are running?
Powershell 7.4.x
PnP.PowerShell 2.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
We dont modify any date values in the code, must be something else in the code/script which must be doing this.

**Reply by Sharepoint2k22** [reply]
Okay thanks, the code is as you see it, e.g. Add-PnPFile -Path $localPath -Folder $Library -Values @{"Modified" = "05/28/2014 16:54"}

Think it might be to do with the locale on the target, lets close this off.

Thanks again.
